string command
string argument
---
string response